generators/task-runner/index.js

```diff
@@ -0,0 +1,53 @@
+const yeoman = require('yeoman-generator');
+const yosay = require('yosay');
+const _merge = require('lodash/merge');
+const _includes = require('lodash/includes');
+
+module.exports = yeoman.Base.extend({
+  prompting: {
+    taskrunner() {
+      const self = this;
+
+      this.log(yosay(
+        'Configure Task Runner for Grunt'
```

Comment:
@riteshe63 This should be `Configure Task Runner`. We can't assume it to be Grunt at this stage